jsonnet/kube-thanos/kube-thanos-receive-hashrings.libsonnet

```
@@ -0,0 +1,59 @@
+local receiveConfigDefaults = import 'kube-thanos/kube-thanos-receive-default-params.libsonnet';
+local receive = import 'kube-thanos/kube-thanos-receive.libsonnet';
+
+// These are the defaults for this components configuration.
+// When calling the function to generate the component's manifest,
+// you can pass an object structured like the default to overwrite default values.
+local defaults = {
```

```suggestion
local defaults = receiveConfigDefaults {
```

No need to nest these defaults, I think :)